cg/apps/scoutapi.py

```diff
@@ -52,10 +52,14 @@ def get_genes(self, panel_id: str, version: str=None) -> list:
             panel genes: panel genes list
         """
         gene_panel = self.gene_panel(panel_id=panel_id, version=version)
-        return gene_panel.get('genes')
 
-    def get_cases(self, case_id=None, institute=None, reruns=None, finished=None, causatives=None, research_requested=None,
-          is_research=None, status=None):
+        if gene_panel:
+            return gene_panel.get('genes')
+
+        return None
```

Comment:
Same here: can we raise an exception instead of returning None?

Reply:
removed